script.js

```diff
@@ -1,31 +1,109 @@
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+  {
+    image: './images/proj-4.jpg',
+    title: 'Multi-Post Stories Gain + Glory',
+    languages: ['Ruby', 'HTML', 'CSS', 'PHP'],
+    details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release.",
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+  {
+    image: './images/proj-5.jpg',
+    title: 'Multi-Post Stories Gain + Glory',
+    languages: ['Ruby', 'HTML', 'CSS', 'PHP'],
+    details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release.",
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+  {
+    image: './images/proj-6.jpg',
+    title: 'Multi-Post Stories Gain + Glory',
+    languages: ['Ruby', 'HTML', 'CSS', 'PHP'],
+    details: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release.",
+    liveVersion: 'https://ehmaddd.github.io',
+    linkSource: 'https://github.com/ehmaddd/ehmaddd.github.io',
+  },
+];
+
+let worksUl;
+const modalUl = document.querySelector('#myModal ul');
+
+// Populating Portfolio
+
+const works = document.querySelectorAll('.work-sample');
+for (let a = 0; a < works.length; a += 1) {
+  document.querySelectorAll('.work-sample img')[a].src = projects[a].image;
+  document.querySelectorAll('.work-sample h4')[a].innerHTML = projects[a].title;
+  worksUl = document.querySelectorAll('.work-sample ul');
+
+  for (let b = 0; b < projects[a].languages.length; b += 1) {
+    const li = document.createElement('li');
+    li.appendChild(document.createTextNode(projects[a].languages[b]));
+    li.setAttribute('id', 'element4');
+    worksUl[a].appendChild(li);
+  }
+  // document.querySelectorAll('.work-sample a')[a].href = "project-"+parseInt(a+1);
```

Comment:
- [ ] it would be better to remove the commented code in order to keep the file clean and pro